vowpalwabbit/core/src/reductions/metrics.cc

```diff
@@ -136,6 +136,12 @@ void predict_or_learn(metrics_data& data, T& base, E& ec)
 
 void additional_metrics(VW::workspace& all, VW::metric_sink& sink)
 {
+  sink.set_float("sum_loss", all.sd->sum_loss);
```

Comment:
This is already accessible in Python

https://vowpalwabbit.org/docs/vowpal_wabbit/python/latest/reference/vowpalwabbit.pyvw.pylibvw.html#vowpalwabbit.pyvw.pylibvw.vw.get_sum_loss